docs/user/introduction.asciidoc

```diff
@@ -1,12 +1,160 @@
+image::images/intro-kibana.png[]
+
+[float]
+[[get-data-into-kibana]]
+=== Getting data into {kib}
+
+{kib} is designed to use {es} as a data source. Think of Elasticsearch as the engine that stores
+and processes the data, with {kib} sitting on top.
+
+From the home page, {kib} provides these options for getting data in:
+
+* <<add-sample-data, Add a sample data set>> and take {kib} for a test drive without loading data yourself.
+* Set up a data flow between your app and the Elastic Stack using our built-in tutorials.
+(If a tutorial doesn’t exist for your data, go to the
+{beats-ref}/beats-reference.html[Beats overview] to learn about other data shippers
+in the {beats} family.)
+* Import log data using the file upload feature.
+* {ref}/getting-started-index.html[Index your data] in Elasticsearch.
++
+[role="screenshot"]
+image::images/intro-data-tutorial.png[Ways to get data in from the home page]
+
+
+{kib} uses an
+<<index-patterns, index pattern>> to tell it which {es} indices to explore.
+If you add sample data or run a built-in tutorial, you get an index pattern for free,
+and are good to start exploring. If you load your own data, you can create
+an index pattern in <<management, Management>>.
+
+[float]
+[[explore-and-query]]
+=== Explore & query
+
+Ready to dive into your data? With <<discover, Discover>>, you can explore your data and
+search for hidden insights and relationships. Ask your questions, and then
+narrow the results to just the data you want.
+
+[role="screenshot"]
+image::images/intro-discover.png[]
+
+[float]
+[[visualize-and-analyze]]
+=== Visualize & analyze
+
+A visualization is worth a thousand log lines, and {kib} provides
+many options for showcasing your data. Use <<visualize, Visualize>> to create
+classic line charts, bar graphs, pie charts, histograms, and tables
+(just to name a few).
+
+Once your visualizations are just the way you want,
+use <<dashboard, Dashboard>> to collect them in one place. A dashboard provides
+insights into your data from multiple perspectives, and you can watch the data
+change in real time.
+
+[role="screenshot"]
+image::images/intro-dashboard.png[]
+
+{kib} also offers these features that address specific visualization needs.
+
+* <<lens, Lens>> provides a drag-and-drop experience that helps
+you get insights from your data in just a few clicks. If there is a
+better visualization for your data, Lens suggests it, allowing
+for quick switching between visualization types.
+
+* <<canvas, Canvas>> gives you the ability to present your data in a
+visually compelling, pixel-perfect report. Give your data the “wow” factor
+needed to impress your CEO or to captivate people with a big-screen display.
+
+* <<maps, Elastic Maps>> enables you to ask (and answer) meaningful
+questions of your location-based data. Elastic Maps supports multiple
+layers and data sources, mapping of individual geo points and shapes,
+and dynamic client-side styling.
+
+* <<TSVB, TSVB>> allows you to combine
```

Comment:
I know "Time Series Visual Builder" is a mouthful, but are we really externally marketing this as TSVB now? I had thought it was just internal

Reply:
@alexfrancoeur there was an effort last year to [change all instances of Time Series Visual Builder to TSVB](https://github.com/elastic/kibana/pull/39125).

Reply:
Ah interesting, naming is hard 😄